specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMaker.json

```diff
@@ -671,6 +679,32 @@
         }
       }
     },
+    "UpdateContextDTO": {
+      "type": "object",
+      "description": "Update Body schema to represent context to be updated",
+      "additionalProperties": false,
+      "properties": {
+        "promptsToDelete": {
+          "type": "array",
+          "description": "List of prompts associated with qna to be deleted",
+          "items": {
+            "type": "integer",
+            "format": "int32"
+          }
+        },
+        "promptsToAdd": {
```

Comment:
It looks like some properties are shared between `UpdateContextDTO` and `ContextDTO`

Reply:
Is this resolved? I don't see any corresponding changes

Reply:
I added comment not sure why it got deleted. There is only one bool property in common rest all are different.